### ISSUE TITLE
Add rich-text support for feedback session description

### ISSUE THREAD

**Issue by belyabl9** [TEAMMATES/teammates#5682]
$subj

As rich-text is based on HTML, we need to disable sanitizing for this field. Is it OK to push such initial version to the master until the new module for preventing XSS, etc. is implemented ? Or it's worth additional branch ?

**Comment by kanghj** [comment]
Seems we missed your query. But if we remove escaping html to support this new feature, then I think we should put it in a separate branch until we can handle sanitization properly.

**Comment by belyabl9** [comment]
@kanghj , I asked similar question via slack and @damithc answered:
"Rich text support for session description alone is releasable". 
May be sanitization removal wasn't considered.
By the way, could you explain required actions to create a separate branch in the main repo ? Am I allowed to create it on my own ?

**Comment by belyabl9** [comment]
This pull request is dependent on #5681 , so tests and required changes (if any) will be done after it's merged

**Comment by kanghj** [comment]
@belyabl9, I am not sure. @wkurniawan07, @damithc, how is @belyabl9 expected to proceed with this issue?

**Comment by damithc** [comment]
Yes, it's not releasable if it's not secure. However, we can add rich text support fully (including sanitization) to session description, release it, and then move to adding the same in other text fields.

**Comment by belyabl9** [comment]
@damithc , ok, then this PR will be frozen until I integrate OWASP. For the moment, I can't estimate how much time it will take because I didn't work with it previously. I will use sanitization rules for initial version as I described in proposal

**Comment by damithc** [comment]
Alright, have a first look and brief us in a day or so. We can decide how to proceed based on your initial estimate of how difficult it's going to be. 
Note that we can make the rich-text support quite restrictive to begin with (i.e. only allow the formattings we are sure that are secure) and then support more things incrementally.
